src/Queue/CoverUserUploadProcessor.php

```diff
@@ -26,13 +26,13 @@
 /**
  * Class CoverUploadProcessor.
  */
-class CoverUploadProcessor implements Processor, TopicSubscriberInterface
+class CoverUserUploadProcessor implements Processor, TopicSubscriberInterface
 {
     private $em;
     private $dispatcher;
     private $statsLogger;
 
-    private const VENDOR_ID = 12;
+    private const VENDOR_ID = 15;
```

Comment:
`VENDOR_ID` Should NOT be hardcoded here!

Reply:
```
Information for Service "App\Service\VendorService\UserUpload\UserUploadVendorService"
======================================================================================

 Class UserUploadVendorService.

 ---------------- -------------------------------------------------------------- 
  Option           Value                                                         
 ---------------- -------------------------------------------------------------- 
  Service ID       App\Service\VendorService\UserUpload\UserUploadVendorService  
  Class            App\Service\VendorService\UserUpload\UserUploadVendorService  
  Tags             app.vendor.service                                            
  Public           no                                                            
  Synthetic        no                                                            
  Lazy             no                                                            
  Shared           yes                                                           
  Abstract         no                                                            
  Autowired        yes                                                           
  Autoconfigured   yes                                                           
 ---------------- -------------------------------------------------------------- 
```

We use DI. Just inject the `UserUploadVendorService` in the constructor and call `getVendorId()`

Reply:
Fixed